milestones/__init__.py

```diff
@@ -3,4 +3,4 @@
 """
 
 
-__version__ = '0.3.3'
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.4.0'
```

Reply:
According to Ned, semantic versioning means that we shouldn't bump to version `1.0.0` based on the upgrade changes.